Can we get a version of SqlCLient released that doesnt have CVEs

Building the following
```
<Project Sdk="Microsoft.NET.Sdk">

    <PropertyGroup>
        <OutputType>Exe</OutputType>
        <TargetFramework>net8.0</TargetFramework>
        <RuntimeIdentifier>win-x64</RuntimeIdentifier>
        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
        <NuGetAuditMode>all</NuGetAuditMode>
        <NuGetAuditLevel>low</NuGetAuditLevel>
    </PropertyGroup>

    <ItemGroup>
      <PackageReference Include="Microsoft.Data.SqlClient" Version="5.2.0" />
    </ItemGroup>

</Project>
```

Reports 3 CVEs

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1903	Warning As Error: Package 'System.Private.Uri' 4.3.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-xhfc-gr8f-ffwc	ConsoleApp1	C:\Code\ConsoleApp1\ConsoleApp1\ConsoleApp1.csproj	1	
Error	NU1902	Warning As Error: Package 'System.Private.Uri' 4.3.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-x5qj-9vmx-7g6g	ConsoleApp1	C:\Code\ConsoleApp1\ConsoleApp1\ConsoleApp1.csproj	1	
Error	NU1903	Warning As Error: Package 'System.Private.Uri' 4.3.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-5f2m-466j-3848	ConsoleApp1	C:\Code\ConsoleApp1\ConsoleApp1\ConsoleApp1.csproj	1	
```

Yes i know consumers can work around these. but given these CVEs date back to May 2022, people should not need to work around them

@JRahnama i must be holding it wrong??

from my local machine

```
PS C:\Code\SqlCve> dotnet nuget locals all --clear
Clearing NuGet HTTP cache: C:\Users\SimonCropp\AppData\Local\NuGet\v3-cache
Clearing NuGet global packages folder: C:\Users\SimonCropp\.nuget\packages\
Clearing NuGet Temp cache: C:\Users\SimonCropp\AppData\Local\Temp\NuGetScratch
Clearing NuGet plugins cache: C:\Users\SimonCropp\AppData\Local\NuGet\plugins-cache
Local resources cleared.
PS C:\Code\SqlCve> dotnet restore
  Determining projects to restore...
C:\Code\SqlCve\ConsoleApp1\ConsoleApp1.csproj : error NU1903: Warning As Error: Package 'System.Private.Uri' 4.3.0 has
a known high severity vulnerability, https://github.com/advisories/GHSA-5f2m-466j-3848 [C:\Code\SqlCve\ConsoleApp1.sln]
C:\Code\SqlCve\ConsoleApp1\ConsoleApp1.csproj : error NU1902: Warning As Error: Package 'System.Private.Uri' 4.3.0 has
a known moderate severity vulnerability, https://github.com/advisories/GHSA-x5qj-9vmx-7g6g [C:\Code\SqlCve\ConsoleApp1.
sln]
C:\Code\SqlCve\ConsoleApp1\ConsoleApp1.csproj : error NU1903: Warning As Error: Package 'System.Private.Uri' 4.3.0 has
a known high severity vulnerability, https://github.com/advisories/GHSA-xhfc-gr8f-ffwc [C:\Code\SqlCve\ConsoleApp1.sln]
  Failed to restore C:\Code\SqlCve\ConsoleApp1\ConsoleApp1.csproj (in 7.58 sec).
PS C:\Code\SqlCve>
```

from a build server (in this case appveyor)

```
Build started
git clone -q --branch=main https://github.com/SimonCropp/SqlCve.git C:\projects\sqlcve
git checkout -qf 20c4f1e9a215cf9c562c963bdab92137fad9dd56
Invoke-WebRequest "https://dot.net/v1/dotnet-install.ps1" -OutFile "./dotnet-install.ps1"
./dotnet-install.ps1 -JSonFile global.json -Architecture x64 -InstallDir 'C:\Program Files\dotnet'
dotnet-install: Remote file https://dotnetcli.azureedge.net/dotnet/Sdk/8.0.202/dotnet-sdk-8.0.202-win-x64.zip size is 294604293 bytes.
dotnet-install: Downloaded file https://dotnetcli.azureedge.net/dotnet/Sdk/8.0.202/dotnet-sdk-8.0.202-win-x64.zip size is 294604293 bytes.
dotnet-install: The remote and local file sizes are equal.
dotnet-install: Extracting the archive.
dotnet-install: Note that the script does not resolve dependencies during installation.
dotnet-install: To check the list of dependencies, go to https://learn.microsoft.com/dotnet/core/install/windows#dependencies
dotnet-install: Installed version is 8.0.202
dotnet-install: Installation finished
dotnet nuget locals all --clear
Clearing NuGet HTTP cache: C:\Users\appveyor\AppData\Local\NuGet\v3-cache
Clearing NuGet global packages folder: C:\Users\appveyor\.nuget\packages\
Clearing NuGet Temp cache: C:\Users\appveyor\AppData\Local\Temp\1\NuGetScratch
Clearing NuGet plugins cache: C:\Users\appveyor\AppData\Local\NuGet\plugins-cache
Local resources cleared.
dotnet build
MSBuild version 17.9.6+a4ecab324 for .NET
  Determining projects to restore...
C:\projects\sqlcve\ConsoleApp1\ConsoleApp1.csproj : error NU1903: Warning As Error: Package 'System.Private.Uri' 4.3.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-5f2m-466j-3848 [C:\projects\sqlcve\ConsoleApp1.sln]
C:\projects\sqlcve\ConsoleApp1\ConsoleApp1.csproj : error NU1902: Warning As Error: Package 'System.Private.Uri' 4.3.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-x5qj-9vmx-7g6g [C:\projects\sqlcve\ConsoleApp1.sln]
C:\projects\sqlcve\ConsoleApp1\ConsoleApp1.csproj : error NU1903: Warning As Error: Package 'System.Private.Uri' 4.3.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-xhfc-gr8f-ffwc [C:\projects\sqlcve\ConsoleApp1.sln]
  Failed to restore C:\projects\sqlcve\ConsoleApp1\ConsoleApp1.csproj (in 10.46 sec).
Build FAILED.
C:\projects\sqlcve\ConsoleApp1\ConsoleApp1.csproj : error NU1903: Warning As Error: Package 'System.Private.Uri' 4.3.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-5f2m-466j-3848 [C:\projects\sqlcve\ConsoleApp1.sln]
C:\projects\sqlcve\ConsoleApp1\ConsoleApp1.csproj : error NU1902: Warning As Error: Package 'System.Private.Uri' 4.3.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-x5qj-9vmx-7g6g [C:\projects\sqlcve\ConsoleApp1.sln]
C:\projects\sqlcve\ConsoleApp1\ConsoleApp1.csproj : error NU1903: Warning As Error: Package 'System.Private.Uri' 4.3.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-xhfc-gr8f-ffwc [C:\projects\sqlcve\ConsoleApp1.sln]
    0 Warning(s)
    3 Error(s)
Time Elapsed 00:00:11.74
Command exited with code 1
```


here is my code: https://github.com/SimonCropp/SqlCve

and same from github action https://github.com/SimonCropp/SqlCve/actions/runs/8415007230/job/23039533364

```
Run dotnet restore
  
  Determining projects to restore...
/home/runner/work/SqlCve/SqlCve/ConsoleApp1/ConsoleApp1.csproj : error NU1903: Warning As Error: Package 'System.Private.Uri' 4.3.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-5f2m-466j-3848 [/home/runner/work/SqlCve/SqlCve/ConsoleApp1.sln]
/home/runner/work/SqlCve/SqlCve/ConsoleApp1/ConsoleApp1.csproj : error NU1902: Warning As Error: Package 'System.Private.Uri' 4.3.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-x5qj-9vmx-7g6g [/home/runner/work/SqlCve/SqlCve/ConsoleApp1.sln]
/home/runner/work/SqlCve/SqlCve/ConsoleApp1/ConsoleApp1.csproj : error NU1903: Warning As Error: Package 'System.Private.Uri' 4.3.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-xhfc-gr8f-ffwc [/home/runner/work/SqlCve/SqlCve/ConsoleApp1.sln]
  Failed to restore /home/runner/work/SqlCve/SqlCve/ConsoleApp1/ConsoleApp1.csproj (in 2.17 sec).
Error: Process completed with exit code 1.
```

--- a/tools/props/Versions.props
+++ b/tools/props/Versions.props
@@ -29,8 +29,8 @@
   <PropertyGroup>
     <AzureIdentityVersion>1.10.3</AzureIdentityVersion>
     <MicrosoftIdentityClientVersion>4.56.0</MicrosoftIdentityClientVersion>
-    <MicrosoftIdentityModelProtocolsOpenIdConnectVersion>7.4.1</MicrosoftIdentityModelProtocolsOpenIdConnectVersion>
-    <MicrosoftIdentityModelJsonWebTokensVersion>7.4.1</MicrosoftIdentityModelJsonWebTokensVersion>
+    <MicrosoftIdentityModelProtocolsOpenIdConnectVersion>7.5.0</MicrosoftIdentityModelProtocolsOpenIdConnectVersion>
+    <MicrosoftIdentityModelJsonWebTokensVersion>7.5.0</MicrosoftIdentityModelJsonWebTokensVersion>
     <SystemBuffersVersion>4.5.1</SystemBuffersVersion>
     <SystemTextEncodingsWebVersion>6.0.0</SystemTextEncodingsWebVersion>
     <MicrosoftSourceLinkGitHubVersion>8.0.0</MicrosoftSourceLinkGitHubVersion>

--- a/tools/props/Versions.props
+++ b/tools/props/Versions.props
@@ -29,8 +29,8 @@
   <PropertyGroup>
     <AzureIdentityVersion>1.10.3</AzureIdentityVersion>
     <MicrosoftIdentityClientVersion>4.56.0</MicrosoftIdentityClientVersion>
-    <MicrosoftIdentityModelProtocolsOpenIdConnectVersion>6.35.0</MicrosoftIdentityModelProtocolsOpenIdConnectVersion>
-    <MicrosoftIdentityModelJsonWebTokensVersion>6.35.0</MicrosoftIdentityModelJsonWebTokensVersion>
+    <MicrosoftIdentityModelProtocolsOpenIdConnectVersion>7.4.1</MicrosoftIdentityModelProtocolsOpenIdConnectVersion>
+    <MicrosoftIdentityModelJsonWebTokensVersion>7.4.1</MicrosoftIdentityModelJsonWebTokensVersion>
     <SystemBuffersVersion>4.5.1</SystemBuffersVersion>
     <SystemTextEncodingsWebVersion>6.0.0</SystemTextEncodingsWebVersion>
     <MicrosoftSourceLinkGitHubVersion>8.0.0</MicrosoftSourceLinkGitHubVersion>

--- a/tools/specs/Microsoft.Data.SqlClient.nuspec
+++ b/tools/specs/Microsoft.Data.SqlClient.nuspec
@@ -31,8 +31,8 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.Data.SqlClient.SNI" version="5.2.0" />
         <dependency id="Azure.Identity" version="1.10.3" />
         <dependency id="Microsoft.Identity.Client" version="4.56.0" />
-        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="6.35.0" />
-        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="6.35.0" />
+        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.4.1" />
+        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.4.1" />
         <dependency id="System.Buffers" version="4.5.1" />
         <dependency id="System.Configuration.ConfigurationManager" version="6.0.1" exclude="Compile" />
         <dependency id="System.Runtime.InteropServices.RuntimeInformation" version="4.3.0" />
@@ -42,8 +42,8 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.Data.SqlClient.SNI.runtime" version="5.2.0" exclude="Compile" />
         <dependency id="Azure.Identity" version="1.10.3" />
         <dependency id="Microsoft.Identity.Client" version="4.56.0" exclude="Compile"/>
-        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="6.35.0" />
-        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="6.35.0" />
+        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.4.1" />
+        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.4.1" />
         <dependency id="Microsoft.SqlServer.Server" version="1.0.0"/>
         <dependency id="System.Configuration.ConfigurationManager" version="8.0.0" exclude="Compile" />
         <dependency id="System.Runtime.Caching" version="8.0.0" exclude="Compile" />
@@ -52,8 +52,8 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.Data.SqlClient.SNI.runtime" version="5.2.0" exclude="Compile" />
         <dependency id="Azure.Identity" version="1.10.3" />
         <dependency id="Microsoft.Identity.Client" version="4.56.0" exclude="Compile"/>
-        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="6.35.0" />
-        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="6.35.0" />
+        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.4.1" />
+        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.4.1" />
         <dependency id="Microsoft.SqlServer.Server" version="1.0.0"/>
         <dependency id="System.Configuration.ConfigurationManager" version="6.0.1" exclude="Compile" />
         <dependency id="System.Runtime.Caching" version="6.0.0" exclude="Compile" />
@@ -62,8 +62,8 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.Data.SqlClient.SNI.runtime" version="5.2.0" exclude="Compile" />
         <dependency id="Azure.Identity" version="1.10.3" />
         <dependency id="Microsoft.Identity.Client" version="4.56.0" exclude="Compile"/>
-        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="6.35.0" />
-        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="6.35.0" />
+        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.4.1" />
+        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.4.1" />
         <dependency id="Microsoft.SqlServer.Server" version="1.0.0"/>
         <dependency id="Microsoft.Win32.Registry" version="5.0.0" exclude="Compile" />
         <dependency id="System.Buffers" version="4.5.1" />
@@ -80,8 +80,8 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.Data.SqlClient.SNI.runtime" version="5.2.0" exclude="Compile" />
         <dependency id="Azure.Identity" version="1.10.3" />
         <dependency id="Microsoft.Identity.Client" version="4.56.0" exclude="Compile"/>
-        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="6.35.0" />
-        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="6.35.0" />
+        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.4.1" />
+        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.4.1" />
         <dependency id="Microsoft.SqlServer.Server" version="1.0.0"/>
         <dependency id="Microsoft.Win32.Registry" version="5.0.0" exclude="Compile" />
         <dependency id="System.Configuration.ConfigurationManager" version="6.0.1" exclude="Compile" />

--- a/tools/specs/Microsoft.Data.SqlClient.nuspec
+++ b/tools/specs/Microsoft.Data.SqlClient.nuspec
@@ -31,8 +31,8 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.Data.SqlClient.SNI" version="5.2.0" />
         <dependency id="Azure.Identity" version="1.10.3" />
         <dependency id="Microsoft.Identity.Client" version="4.56.0" />
-        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.4.1" />
-        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.4.1" />
+        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.5.0" />
+        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.5.0" />
         <dependency id="System.Buffers" version="4.5.1" />
         <dependency id="System.Configuration.ConfigurationManager" version="6.0.1" exclude="Compile" />
         <dependency id="System.Runtime.InteropServices.RuntimeInformation" version="4.3.0" />
@@ -42,8 +42,8 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.Data.SqlClient.SNI.runtime" version="5.2.0" exclude="Compile" />
         <dependency id="Azure.Identity" version="1.10.3" />
         <dependency id="Microsoft.Identity.Client" version="4.56.0" exclude="Compile"/>
-        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.4.1" />
-        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.4.1" />
+        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.5.0" />
+        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.5.0" />
         <dependency id="Microsoft.SqlServer.Server" version="1.0.0"/>
         <dependency id="System.Configuration.ConfigurationManager" version="8.0.0" exclude="Compile" />
         <dependency id="System.Runtime.Caching" version="8.0.0" exclude="Compile" />
@@ -52,8 +52,8 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.Data.SqlClient.SNI.runtime" version="5.2.0" exclude="Compile" />
         <dependency id="Azure.Identity" version="1.10.3" />
         <dependency id="Microsoft.Identity.Client" version="4.56.0" exclude="Compile"/>
-        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.4.1" />
-        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.4.1" />
+        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.5.0" />
+        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.5.0" />
         <dependency id="Microsoft.SqlServer.Server" version="1.0.0"/>
         <dependency id="System.Configuration.ConfigurationManager" version="6.0.1" exclude="Compile" />
         <dependency id="System.Runtime.Caching" version="6.0.0" exclude="Compile" />
@@ -62,8 +62,8 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.Data.SqlClient.SNI.runtime" version="5.2.0" exclude="Compile" />
         <dependency id="Azure.Identity" version="1.10.3" />
         <dependency id="Microsoft.Identity.Client" version="4.56.0" exclude="Compile"/>
-        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.4.1" />
-        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.4.1" />
+        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.5.0" />
+        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.5.0" />
         <dependency id="Microsoft.SqlServer.Server" version="1.0.0"/>
         <dependency id="Microsoft.Win32.Registry" version="5.0.0" exclude="Compile" />
         <dependency id="System.Buffers" version="4.5.1" />
@@ -80,8 +80,8 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.Data.SqlClient.SNI.runtime" version="5.2.0" exclude="Compile" />
         <dependency id="Azure.Identity" version="1.10.3" />
         <dependency id="Microsoft.Identity.Client" version="4.56.0" exclude="Compile"/>
-        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.4.1" />
-        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.4.1" />
+        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="7.5.0" />
+        <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.5.0" />
         <dependency id="Microsoft.SqlServer.Server" version="1.0.0"/>
         <dependency id="Microsoft.Win32.Registry" version="5.0.0" exclude="Compile" />
         <dependency id="System.Configuration.ConfigurationManager" version="6.0.1" exclude="Compile" />

